velox/vector/tests/EnsureWritableVectorTest.cpp

```diff
@@ -781,7 +781,11 @@ TEST_F(EnsureWritableVectorTest, allNullMap) {
       /*sizeAt*/ [](vector_size_t row) { return row % 5; },
       /*keyAt*/ [](vector_size_t row) { return row; },
       /*valueAt*/
-      [](vector_size_t row) { return StringView("s-" + std::to_string(row)); },
+      [](vector_size_t row) {
+        auto str = "s-" + std::to_string(row);
+        VELOX_CHECK(StringView::isInline(str.size()));
+        return StringView(str);
```

Comment:
this looks like a super common pattern. I wonder if we should allow people to do something like:

```
StringView::makeInline("s-" + std::to_string(row));
```

or similar API.